### DISCUSSION TITLE
separate taking screenshots and generating html in snapshot

### DISCUSSION THREAD

**Discussion by ojii** [fastlane/fastlane#18329] [Q&A]
I'm using snapshot to automatically take screenshots of my iOS app, but the process is incredibly slow on CI since I take screenshots in 4 different configurations (3 languages * 4 devices). I'm using Circle CI which allows running things in parallel on different machines which I could use to speed this up, but that would generate multiple html pages of screenshots and ideally I'd keep the single page.

Is there a way to instruct snapshot to skip generating the html page and then later direct it at a directory of snapshots to build the html page of?

**Comment by dinsen** [comment]
Hi @ojii 
Can you show your Snapfile or how you current configuration is?

**Reply by ojii** [reply]
sure:
```
devices([
   "iPhone 12 Pro Max",
   "iPhone 8 Plus",
   "iPad Pro (12.9-inch) (4th generation)",
   "iPad Pro (12.9-inch) (2nd generation)",
])

languages([
  "en",
  "ja",
  "zh-Hant-TW"
])

scheme("Screenshots")
output_directory("./screenshots")
clear_previous_screenshots(true)
override_status_bar(true)
headless(true)
```
